joomla/joomla-cms
# [5.2] Fix public folder path when trailing slash is added

### Summary of Changes
Remove any trailing slashes for public folder path constant


### Testing Instructions
- Install a Joomla! 5.2 via the console using a public folder (description here: https://github.com/joomla/joomla-cms/pull/41446 )
- make sure you add a slash at the end of your public folder path
- finish installation
- check e.g. if the scheduler ajax call works (via the browser console)


### Actual result BEFORE applying this Pull Request
- error: Site not found


### Expected result AFTER applying this Pull Request
- ajax call works
- no trailing slash in the defines.php in your public folder

@dgrammatiko probably you could check if you agree

thanks @HLeithner for helping debugging.
